llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -285,13 +285,19 @@ def RSqrt : DXILOpMapping<25, unary, int_dx_rsqrt,
                          "Returns the reciprocal of the square root of the specified value."
                          "rsqrt(x) = 1 / sqrt(x).",
                          [llvm_halforfloat_ty, LLVMMatchType<0>]>;
+def Rbits : DXILOpMapping<30, unary, int_bitreverse,
+                         "Returns the specified value with its bits reversed.",
+                         [llvm_anyint_ty, LLVMMatchType<0>]>;
```

Comment:
```suggestion
                         "Returns the specified value with its bits reversed.">;
```

Since the inherited types of `int_bitreverse` are valid for `Rbits`, overriding is not necessary,

Additionally, consider adding this record after `Trunc` to maintain ascending order of opcode value.

Reply:
@bharadwajy i'm pretty sure llvm_anyint_ty also covers int vectors. don't we need to limit to int scalars?

Reply:
> @bharadwajy i'm pretty sure llvm_anyint_ty also covers int vectors. don't we need to limit to int scalars?

Yes. At present, such a narrower scalar type is not yet defined in `DXIL.td`. Additional refinement will follow once a way to represent the narrower overload type is added.

Reply:
Does it make sense to do the narrower scalar type in this pr?  Alternatively, should we leave this as is, as a placeholder for the future refinement?

Reply:
> Does it make sense to do the narrower scalar type in this pr? Alternatively, should we leave this as is, as a placeholder for the future refinement?

Merging this PR in its current form and following up with a refinement in a later PR would be beneficial. That would follow the path of continued addition of DXIL Ops and tests for lowering, even if overload specification includes vector types to inform my ongoing design space exploration of a better specification mechanism, without being blocked on either front.